### DISCUSSION TITLE
Nested namespaces

### DISCUSSION THREAD

**Discussion by lmilbaum** [community/community#45611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Gitlab provides a nested namespaces capability. It is super helpful for managing a large amount of repositories in one organization. For example: managing collaborators privileges per a group of repositories, acting on all open PRs, etc..
Is that something that GitHub will support?

**Comment by jge162** [comment]
Hello @lmilbaum Thank you for your post, GitHub does not have a built-in nested namespace feature, it does provide other tools and features that can be used to manage large numbers of repositories and collaborators within an organization.

[GitHub Organization and Teams documentation](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams)

[GitHub Actions](https://docs.github.com/en/actions)

**Reply by lmilbaum** [reply]
Hello @jge162 and thank you for your prompt feedback.  i'm aware of those features. The DevEx is not as intuitive as the nested namespace feature.

**Reply by jge162** [reply]
> Hello @jge162 and thank you for your prompt feedback. i'm aware of those features. The DevEx is not as intuitive as the nested namespace feature.

agreed, thanks for your post as well.

**Comment by drmuey** [comment]
We’d vote for this to. Teams are great and consistent prefixes help but in a large org w/ many projects one flat list of repos is untenable. Doesn’t need to be multi level just 

* /org/product1
   * repo 1
   * repo 2
   * …
* /org/product2
   * repo 3
   * repo 4
   * …
   
   Maybe a feature request instead of discussion?

**Comment by RenaudDenis** [comment]
+1
